IntegrationTests/Sources/IntegrationTestSupport/Helpers.swift

```diff
@@ -138,14 +136,17 @@ func sh(
     let stderr = try result.utf8stderrOutput()
 
     if result.exitStatus != .terminated(code: 0) {
-        XCTFail("Command failed with exit code: \(result.exitStatus) - \(result.integrationTests_debugDescription)", file: file, line: line)
+        XCTFail(
```

Comment:
issue (blocking): Since we convert the Integration Tests to Swift Testing, `XCTFail` will not work, or may not properly report the failure message.

Can we use `Issue.record(...)` instead?  See the [Migration from XCTest](https://developer.apple.com/documentation/testing/migratingfromxctest) documentation..

Reply:
Yes, `Issue.record` is the canonical replacement for `XCTFail`.

Never mix XCTest and Swift Testing APIs in the same test.